web/package.json

```diff
@@ -5,48 +5,35 @@
   "license": "MIT",
   "scripts": {
     "data": "cd .. && python3 scripts/convert_to_web_app_json.py",
-    "clear": "node ./lib/clear.js",
     "dev": "nodemon --config config/nodemon/dev.json",
     "dev:start": "yarn install && yarn dev:nowatch || cd .",
-    "dev:nowatch": "cross-env NODE_ENV=development BABEL_ENV=development NODE_OPTIONS=--max_old_space_size=5000  babel-node --config-file \"./babel-node.config.js\" --extensions \".js,.ts\" -- node_modules/.bin/next dev --hostname 0.0.0.0 --port 3000",
-    "dev:start_win": "yarn install && yarn dev:nowatch_win || cd .",
-    "dev:nowatch_win": "cross-env NODE_ENV=development BABEL_ENV=development NODE_OPTIONS=--max_old_space_size=5000  babel-node --config-file \"./babel-node.config.js\" --extensions \".js,.ts\" -- ./node_modules/next/dist/bin/next dev --hostname 0.0.0.0 --port 3000",
-    "dev:clean": "rimraf '.build/development/{*,.*}' 'node_modules/.cache'",
-    "prod:build": "yarn run next:build && yarn run next:export",
-    "prod:build:ci": "yarn run prod:build",
+    "dev:nowatch": "cross-env NODE_ENV=development BABEL_ENV=development next dev --port 3000",
```

Comment:
Do we really need `dev`, `dev:start` and `dev:nowatch`?

Reply:
For dev, I like having the nodemon watch because it triggers on more files than next's hot reload (e.g. when adding packages). Inlining `dev:nowatch` is nice now that the command is shorter.